.github/CONTRIBUTING.md

```diff
@@ -70,6 +70,42 @@ Before we can accept your pull requests you'll need to sign a Contributor Licens
 
 You can sign these electronically (just scroll to the bottom). After that, we'll be able to accept your pull requests.
 
+### Publishing
+
+To publish this module, you must be logged in as the user `google-cloud`.
+
+```sh
+$ npm logout && npm login
+$ npm login
+# Follow the prompts to login as `google-cloud`
+
+# If publishing a service module:
+#   $ cd packages/bigquery
+# Otherwise stay in the root directory
+
+$ npm run publish-module {{version}}
+```
+
+  - **version** In the format of `0.2.0`
+
+This spawns these commands (**do not run these manually!**):
```

Comment:
If we don't want some one to run them manually, maybe we should give a brief summary of what the command does instead of listing the exact commands?